modules/devguide/examples/ConnectionBase.java

```diff
@@ -14,15 +14,14 @@
  * limitations under the License.
  */
 
-package com.couchbase.devguide;
```

Comment:
Why are you removing the `package` declaration in this and subsequent commits?

Reply:
Since I'm moving these to use the root pom file and essentially being part of the package defined there it no longer seems necessary to separate these examples into their own package.